backend/src/routes/RestaurantRoute.ts

```diff
@@ -10,6 +10,18 @@ router.get(
   param('city').isString().trim().notEmpty().withMessage('City is required'),
   jwtCheck,
   jwtParse,
+  RestaurantController.getRestaurant,
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/45#issuecomment-2411028139>#43cd30</a><span> - 10/14/2024, 12:06 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The controller method for the '/search/:city' route has been changed to RestaurantController.getRestaurant, which is likely incorrect for searching restaurants by city.
</div>
<div id="fix">
<b>Fix:</b> Replace RestaurantController.getRestaurant with RestaurantController.searchRestaurants for the '/search/:city' route.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -13,1 +13,1 @@
 -  RestaurantController.getRestaurant,
 +  RestaurantController.searchRestaurants,
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged